docs/reference/http-server.md

```
@@ -666,16 +673,79 @@ declared type is the same whether or not a field is pinned: `Env` and
+  router: orderRouter,
+  imports: [OrdersSlice, CustomersSlice],
+  cors: { origin: "https://orders.example", credentials: true },
+  bodyLimit: 5_000_000,
+  compression: true,
+});
+```
+
+`true` takes the underlying plugin's own defaults; a record is that plugin's
+own options type verbatim, never a `Record<string, unknown>` bag.
+
+**The scalar half of each is a field of [`HttpConfig`](#httpconfig), and the
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)